scipy/optimize/_optimize.py

```diff
@@ -1732,7 +1741,7 @@ def fmin_cg(f, x0, fprime=None, args=(), gtol=1e-5, norm=np.inf,
 def _minimize_cg(fun, x0, args=(), jac=None, callback=None,
                  gtol=1e-5, norm=np.inf, eps=_epsilon, maxiter=None,
                  disp=False, return_all=False, finite_diff_rel_step=None,
-                 **unknown_options):
+                 c1=1e-3, c2=.4, **unknown_options):
```

Comment:
Where are you getting these values from? Nocedal recommends `1e-4` and `0.1` for CG in [Numerical Optimization](https://link.springer.com/book/10.1007/978-0-387-40065-5)

Reply:
I just looked very quickly and I can't see `c1=1e-3` used anywhere. From [a rapid search](https://github.com/search?q=repo%3Ascipy%2Fscipy+c1%3D1e&type=code), it seems like it was `c1=1e-4`.  I think somebody else should double check this, as changing these values might break backward compatibility.

Reply:
Thanks yes this looks like a typo on my behalf